interface/patient_file/encounter/superbill_custom_full.php

```diff
@@ -58,7 +58,7 @@ function bucks($amount)
     $modifier   = $_POST['modifier'];
     $superbill  = $_POST['form_superbill'];
     $related_code = $_POST['related_code'];
-    $cyp_factor = $_POST['cyp_factor'] + 0;
+    $cyp_factor = is_numeric($_POST['cyp_factor']) ? $_POST['cyp_factor'] + 0 : 0;
```

Comment:
as discussed above, this is a good format for the fixes above (unless I am missing something obvious which I frequently do)

Reply:
For me this is unknown numeric so, while reasonable in the first place, this allows code down the line determine numeric type.